app/controllers/reports_controller.rb

```diff
@@ -75,18 +75,10 @@ def destroy
   end
 
   def send_report_csv
-    # override DEFAULT_PARAMS of report_helper.rb to get raw data
-    # TO DO: add flag to turn off thresholding rather than editing params
-    params[:threshold_zscore] = 0
-    params[:threshold_rpm] = 0
-    params[:threshold_r] = 0
-    params[:threshold_percentidentity] = 0
-    params[:threshold_alignmentlength] = 0
-    params[:threshold_neglogevalue] = 0
-    params[:threshold_aggregatescore] = 0
-    params[:excluded_categories] = 'None'
-    # serve CSV
-    @report_csv = generate_report_csv(external_report_info(@report, params))
+    params[:is_csv] = 1
+    params[:sort_by] = "highest_nt_aggregatescore"
```

Comment:
oh i see... i think it may be fine to put the disable_filters here then, so we don't need to have the check for filters in taxonomy_details

the check for truncating results we still need, or we can make max_rows a URI param and make this perfectly uniform

Reply:
Makes sense. I think we'll redesign the report CSV functionality once the button is back in the frontend and we get some user feedback on whether they mostly want to download raw data or whether they want to be able to filter/truncate/sort... I'll postpone reworking this until then.